src/qibo/base/callbacks.py

```diff
@@ -11,7 +11,8 @@ class Callback(ABC):
 
     Results of a callback can be accessed by indexing the corresponding object.
     """
-    from qibo.config import K
+    # FIXME: Remove this because base should be independent of backends
+    from qibo import K
```

Comment:
Does this still necessary?

Reply:
I removed this and moved some functions that were using it to `core/callbacks.py`.